apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/PromptResourceTest.java

```diff
@@ -0,0 +1,318 @@
+                .newDatabaseAnalyticsFactory(CLICKHOUSE_CONTAINER, DATABASE_NAME);
+
+        app = TestDropwizardAppExtensionUtils.newTestDropwizardAppExtension(
+                MYSQL.getJdbcUrl(), databaseAnalyticsFactory, wireMock.runtimeInfo(), REDIS.getRedisURI());
+    }
+
+    private final PodamFactory factory = PodamFactoryUtils.newPodamFactory();
+
+    private String baseURI;
+    private ClientSupport client;
+
+    @BeforeAll
+    void setUpAll(ClientSupport client, Jdbi jdbi) throws SQLException {
+
+        MigrationUtils.runDbMigration(jdbi, MySQLContainerUtils.migrationParameters());
+
+        try (var connection = CLICKHOUSE_CONTAINER.createConnection("")) {
+            MigrationUtils.runDbMigration(connection, CLICKHOUSE_CHANGELOG_FILE,
+                    ClickHouseContainerUtils.migrationParameters());
+        }
+
+        this.baseURI = "http://localhost:%d".formatted(client.getPort());
+        this.client = client;
+
+        ClientSupportUtils.config(client);
+
+        mockTargetWorkspace(API_KEY, TEST_WORKSPACE, WORKSPACE_ID);
+    }
+
+    private static void mockTargetWorkspace(String apiKey, String workspaceName, String workspaceId) {
+        AuthTestUtils.mockTargetWorkspace(wireMock.server(), apiKey, workspaceName, workspaceId, USER);
+    }
+
+    @AfterAll
+    void tearDownAll() {
+        wireMock.server().stop();
+    }
+
+    @Nested
+    @DisplayName("Api Key Authentication:")
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class ApiKey {
+
+        private final String fakeApikey = UUID.randomUUID().toString();
+        private final String okApikey = UUID.randomUUID().toString();
+
+        Stream<Arguments> credentials() {
+            return Stream.of(
+                    arguments(okApikey, true),
+                    arguments(fakeApikey, false),
+                    arguments("", false));
+        }
+
+        @BeforeEach
+        void setUp() {
+
+            wireMock.server().stubFor(
+                    post(urlPathEqualTo("/opik/auth"))
+                            .withHeader(HttpHeaders.AUTHORIZATION, equalTo(fakeApikey))
+                            .withRequestBody(matchingJsonPath("$.workspaceName", matching(".+")))
+                            .willReturn(WireMock.unauthorized()));
+
+            wireMock.server().stubFor(
+                    post(urlPathEqualTo("/opik/auth"))
+                            .withHeader(HttpHeaders.AUTHORIZATION, equalTo(""))
+                            .withRequestBody(matchingJsonPath("$.workspaceName", matching(".+")))
+                            .willReturn(WireMock.unauthorized()));
+        }
+
+        @ParameterizedTest
+        @MethodSource("credentials")
+        @DisplayName("create prompt: when api key is present, then return proper response")
+        void createProject__whenApiKeyIsPresent__thenReturnProperResponse(String apiKey, boolean success) {
+
+            var project = factory.manufacturePojo(Project.class);
```

Comment:
I guess it should be:
```suggestion
            var prompt = factory.manufacturePojo(Prompt.class);
```

But then the test should probably fail. Am I missing something?

Reply:
Both entities have only name and description fields, and only name is mandatory.